app/services/preserved_object_handler.rb

```diff
@@ -261,19 +271,51 @@ def update_pc_invalid_moab
       pres_copy = PreservedCopy.find_by!(preserved_object: pres_object, endpoint: endpoint) if pres_object
       # FIXME: what if there is more than one associated pres_copy?
       update_status(pres_copy, PreservedCopy::INVALID_MOAB_STATUS)
-      update_pc_audit_timestamps(pres_copy, true, false)
+      update_pc_audit_timestamps(pres_copy, false)
       update_db_object(pres_copy)
     end
     handler_results.remove_db_updated_results unless transaction_ok
   end
 
-  def update_pc_unexpected_version(pres_copy, pres_object, new_status, ran_moab_validation)
+  def check_status_if_not_ok_or_unexpected_version(pres_copy, found_expected_version)
+    # if pres_copy.status != PreservedCopy::OK_STATUS
+    unless pres_copy.status == PreservedCopy::OK_STATUS && found_expected_version
+      set_status_as_seen_on_disk(pres_copy, found_expected_version)
+    end
+  end
+
+  # given a PreservedCopy instance and whether the caller found the expected version of it on disk, this will perform
+  # other validations of what's on disk, and will update the status accordingly
+  # FIXME: this method probably needs to update validation timestamps?  and tests would need to get fixed to reflect that.
+  def set_status_as_seen_on_disk(pres_copy, found_expected_version)
+    # TODO: do the check that'd set PreservedCopy::ONLINE_MOAB_NOT_FOUND_STATUS
+    # TODO: nothing in the code seems to set ONLINE_MOAB_NOT_FOUND_STATUS atm, but everything we're checking right now is M2C and thus found by definition
```

Comment:
I think the above two comments could be combined.  Maybe "TODO: when we implement C2M, this is likely where we'll set status to ..."